pages/[templateId].tsx

```diff
@@ -26,12 +30,9 @@ const NftDetailPage: FC = () => {
     : localizationJson['en'].detailPage;
 
   const router = useRouter();
-  const { collection, templateId } = router.query as RouterQuery;
+  const { templateId } = router.query as RouterQuery;
 
-  const { template, isLoading, error } = useFetchNft({
-    collection: collection ? collection.toLowerCase() : '',
-    templateId,
-  });
+  const { template, isLoading, error } = useFetchNft(templateId);
```

Comment:
Here we should handle a situation where someone enters in a templateId that is not part of this collection, like routing them back to the homepage.